uniffi_bindgen/src/bindings/swift/templates/Template-Bridging-Header.h

```diff
@@ -31,6 +47,7 @@ typedef struct NativeRustError {
     char *_Nullable message;
 } NativeRustError;
 
+#endif // def UNIFFI_SHARED_H
```

Comment:
1. We need to version bump and a release, so I picked a minor release number.
2. I'm not sure to do that here in this PR or in a separate release PR.

Reply:
I support `v0.8.0` and I support doing it in a separate PR 👍